client/src/client.rs

```diff
@@ -1017,6 +1017,21 @@ pub trait RpcApi: Sized {
         self.call("uptime", &[])
     }
 
+    /// Submit a raw block
+    fn submit_block(&self, block: &[u8]) -> Result<String> {
+        let block_hex: String = block.to_hex();
+        self.submit_block_hex(&block_hex)
+    }
+
+    /// Submit a block as a hex string
+    fn submit_block_hex(&self, block_hex: &String) -> Result<String> {
+        match self.call("submitblock", &[into_json(&block_hex)?]) {
+            Ok(serde_json::Value::Null) => Ok("Success".to_string()),
```

Comment:
This should return a `Result<()>` instead of a success string.

Reply:
Addressed in https://github.com/rust-bitcoin/rust-bitcoincore-rpc/pull/151/commits/9340a293b0e14f125145cf404e26204bceaa56a3